### PR TITLE
feat: Introduced .editorconfig file to the project

### LINKED ISSUE #210: feat: adding an editorconfig file to project
### Is your feature request related to a problem? Please describe.
Editor  config file will help to maintain the standard code formatting throughout the project style which will eventually leads to better code readability

### Describe the solution you'd like
Adding .editorconfig file considering the best code standards for the project.

### Describe alternatives you've considered
This feature do not have any alternative adding an editorconfig file

### Additional context
Choosing code standards in such a way that should make code readable and easy to understand.

### PR DESCRIPTION
**Pull request by nlok5923**
### Description

added editorconfig file to project to maintain code standards 

Fixes #210 

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance

**Code/Quality Assurance Only**

- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
If you are working on a design then add a gif to show the responsiveness of your design.
Please follow [testing guidelines](https://github.com/anitab-org/anitab-org.github.io/wiki/Prepare-testing-GitHub-Pages-for-your-PR) and share the link of deployed site here.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by nlok5923** [comment]
@annabauza  please review this pr.

**Comment by nlok5923** [comment]
@nandini45 please can you review this pull request.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (gaurivn) [inline .editorconfig:12]:
```suggestion
trim_trailing_whitespace = false

```

**Review by gaurivn — CHANGES_REQUESTED** [review]
@nlok5923, LGTM, please add a newline at end of file as shown, Thank you

File: .editorconfig (hunk 1 above)
Comment (nlok5923) [reply .editorconfig:12]: Thanks for requesting changes will resolve it in latest commit.

**Review by isabelcosta — APPROVED** [review]
I think this looks good :)
Thank you for your work @nlok5923 🙌🏾

**Review by gaurivn — APPROVED** [review]
Good, Thank you @nlok5923 🎉🎉🎊🎊

**Comment by isabelcosta** [comment]
@nlok5923 @gaurivn just out of curiosity, how will this config file enforce the coding style? Will there be a lint command we can run to make the code follow this?

**Comment by gaurivn** [comment]
@isabelcosta, as far as I know, there is no need to run a lint command specifically for this purpose, the .editorconfig file can be read by IDEs, code editors, Visual Studio, Intellij, Eclipse, etc. to enforce the coding style. On opening the project, the editor will see the .editorconfig file, and override the settings to suit the project's formatting conventions.

**Comment by isabelcosta** [comment]
Amazing! I just learned something new 🤓 Thank you @gaurivn !

**Comment by gaurivn** [comment]
You're welcome @isabelcosta 😃

**Pull request merged**